Assets/World/Factions/Faction.gd

```diff
@@ -0,0 +1,7 @@
+extends Resource
+class_name Faction
+
+
```

Comment:
Remove redundant line as we follow a consistent 1 blank line convention between each block/section of code.
```suggestion

```

Reply:
I don't think we should remove the two line breaks, since the style guide enforces them: https://docs.godotengine.org/en/latest/tutorials/scripting/gdscript/gdscript_styleguide.html

Reply:
The style guide doesn't "enforce" anything and the readability argument is highly subjective. However, I don't mind a permanent switch to 2 lines, but having code styles mixed up is even worse. The entire codebase must be adapted to one style or another at first.

Reply:
Okay, in this case we can keep as it's now, then later I might format all the scripts to comply with the 2 line breaks.